docroot/themes/humsci/humsci_basic/src/scss/components/_vertical-linked-card.scss

```diff
@@ -238,3 +225,11 @@
     }
   }
 }
+
+.hb-stretch-vertical-linked-cards {
+  height: 100%;
+
+  .field-hs-collection-items .ptype-hs-postcard {
+    display: flex;
+  }
+}
```

Comment:
These styles only work for a very specific markup structure. Isn't it possible to have the same effect with a more generic approach? The `hb-stretch-vertical-linked-cards` can be used in other context, such as views.

Reply:
@cienvaras I can check and add a more generic approach. Do you have an example where that's being used in other context? Just to be sure that what I do, works as expected

Reply:
@mariannuar  Don't have a specific example right now, but according to the [Utility class documentation](https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/themes/humsci/humsci_basic/docs/utility-classes.md) it's possible to use this class in views. Imagine something like this:

```
<div class="hb-stretch-vertical-linked-cards">
  // Any markup structure in the middle.
  <div class="hb-vertical-linked-card">
    ...
  </div>
</div>
```

Reply:
Oh ok. Thanks!

Reply:
@cienvaras Oh got it. Thanks :)